src/nodes/NodeManager.ts

```diff
@@ -298,10 +315,10 @@ class NodeManager {
       const claimId = c as ClaimIdString;
       const payload = verifiedChainData[claimId].payload;
       if (payload.data.type === 'node') {
-        const endNodeId = payload.data.node2;
+        const endNodeId = nodesUtils.decodeNodeId(payload.data.node2)!;
```

Comment:
Same as https://github.com/MatrixAI/js-polykey/pull/318#discussion_r793286473. No validation on node ID yet, so this makes an assumption that node2 of the node to node claim is a valid node ID.